packages/playground/cases/react/basic/rspack.config.js

```diff
@@ -11,7 +11,10 @@ module.exports = {
 			}
 		]
 	},
-	entry: ["@rspack/dev-client/react-refresh-entry", "./src/index.jsx"],
```

Comment:
Let's keep this one and create a new test instead until the `dev-client` is fully deprecated.